modules/homefinder/widgets/class-affiliate-search-widget.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * HomeFinder Affiliate Search Widget (http://www.homefinder.com/widgets/)
+ *
+ * @package RE-PRO
+ */
+
+	/* Exit if accessed directly. */
+if ( ! defined( 'ABSPATH' ) ) { exit; }
+
+/**
+ * HomeFinderAffiliateSearch class.
+ *
+ * @extends WP_Widget
+ */
+class HomeFinderAffiliateSearch extends WP_Widget {
+
+	/**
+	 * __construct function.
+	 *
+	 * @access public
+	 * @return void
+	 */
+	public function __construct() {
+
+		parent::__construct(
+			'homefinder_affiliate_search',
+			__( 'HomeFinder Affiliate Search', 're-pro' ),
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->